### ISSUE TITLE
Compiling notice in footer.

### ISSUE THREAD

**Issue by pixeljitsu** [bdkjones/CodeKit2#193]
When compiling sass, the spinner in the bottom right and the text notice of Compiling "file.scss"... does not disappear.  The system notification will show and disappear, but the notice in the footer does not.

**Comment by bdkjones** [comment]
I need the file that's showing this issue along with any files it imports. Please zip them and post a download link here.

**Comment by pixeljitsu** [comment]
https://www.dropbox.com/s/9jsmfmqjg9nr662/scss.zip

**Comment by bdkjones** [comment]
I can't duplicate this over here. I need details about how you're using the app.

Is the folder local on your machine?
What IDE are you saving with?
What OS version?

DETAILS. Bug reports without details are pretty much useless.

**Comment by pixeljitsu** [comment]
The folder is local. I am using Sublime Text 3 on 10.9.1 Build 13B42

The occurrence of this is kinda spotty, not always right away at first, but after using the app long enough it will happen. Even though the message is stuck, additional saves will compile as normal (including the OSX notification), a new log item is added with the green check mark, but the compiling message stays.  Not sure if this is related but sometimes the log message seems to be formatted weird.  I added a png in the same dropbox folder I shared earlier.

If I switch to the Assets tabs the message will change to Scanning for Installed Components... and will then switch back to the stuck compiling message.  I'm not using libsass, but it seems to behave the same way.  Shift + restart does not seem to fix the issue.

Let me know if there is anything I can do when it happens again to give you more info.
